### DISCUSSION TITLE
Problems using the command line data importer

### DISCUSSION THREAD

**Discussion by florian-guillemard** [firefly-iii/firefly-iii#6936] [Firefly III]
Hi,

I'm working on a docker installation, Firefly is working perfectly (nice job guys!).
I have a nginx reverse proxy on docker too.

I put a container for the data importer, it's working well from the web interface (but it is in publlic) with a CLIENT_ID and FIREFLY_III_URL.

I want to execute command lines in order to do automation.

(Maybe there is a better way)

I already did an import with Nordigen from the web but when i try in the docker FIDI container or outside of it, I got an "Unauthenticated"  401.

I tried with an ACCESS_TOKEN too but always the same message :
```
Could not connect to Firefly III at http://firefly:8080: Error accessing "http://firefly:8080/api/v1/about?". Status code is 401. Body is: {"message":"Unauthenticated","exception":"AuthenticationException"}
Could not connect to your local Firefly III instance at http://firefly:8080.
```

Thanks for your help,

**Comment by JC5** [comment]
You have to set the `FIREFLY_III_ACCESS_TOKEN`, either using the command line or the env file.

**Reply by florian-guillemard** [reply]
I already did it...

I manage to change the "app/Console/HaveAccess.php" to show me the access token and it is passed !

For using ACCESS_TOKEN, I have to remove the CLIENT_ID though (the only thing which is working, but in the web version).

**Reply by JC5** [reply]
Ah yes, those two can conflict with each other, sorry about that.

**Reply by florian-guillemard** [reply]
No problem.. I can stop using the web version (with CLIENT_ID) because I want to use the command line version !

I also tried the POST version and same result :(

**Reply by JC5** [reply]
I'll make sure the data importer will warn you about that :+1:

**Reply by florian-guillemard** [reply]
I found my (first) problem.. it appears that ACCESS_TOKEN should be PERSONNAL ACCESS TOKEN 😅
I tried with the COMMAND LINE TOKEN.. Too many token for me 🤣
Thanks for you help @JC5 😘
